src/components/Code/Code.test.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import enzyme, { shallow, mount } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+import { withTheme } from '../../utils/theme';
+import Code from '.';
+
+enzyme.configure({ adapter: new Adapter() });
+
+document.execCommand = () => {};
+document.getSelection = () => ({
+  toString: () => '',
+});
```

Comment:
Do these get properly rolled back after the tests are finished?

Reply:
Great question, I believe they do as each test is isolated.
They are purely there to stop Jest barfing about `document.execCommand` not being a function as it is neither are mocked in JSDom.

Reply:
Just checked and the tests are isolated so this will not affect other tests regardless